src/palace/manager/util/http.py

```diff
@@ -199,10 +255,11 @@ def request_with_timeout(
     @classmethod
     def _request_with_timeout(
         cls,
+        http_method: str,
         url: str,
         make_request_with: Callable[..., Response],
-        *args: Any,
-        **kwargs: Any,
+        process_response_with: _ProcessResponseCallable | None = None,
+        **kwargs: Unpack[RequestKwargs],
     ) -> Response:
```

Comment:
This doesn't get called directly much, its is mostly used within this class, but I changed around the parameter order to be consistent with the other methods. This required some changes in tests.